src/function/macro_function.cpp

```diff
@@ -17,7 +17,7 @@ MacroFunction::MacroFunction(MacroType type) : type(type) {
 }
 
 string MacroFunction::ValidateArguments(MacroFunction &macro_def, const string &name, FunctionExpression &function_expr,
-                                        vector<unique_ptr<ParsedExpression>> &positionals,
+                                        vector<unique_ptr<ParsedExpression>> *positionals,
```

Comment:
I still don't understand why this is a pointer - can we turn it into a reference again? The function will crash if called with `nullptr` as argument.